library/std/src/lib.rs

```diff
@@ -527,6 +520,31 @@ pub mod task {
     pub use alloc::task::*;
 }
 
+#[doc = include_str!("../../stdarch/crates/core_arch/src/core_arch_docs.md")]
```

Comment:
The links in the `Other architectures` section are broken when this documentation is put here in `std`. E.g. `x86` links to `std/arch/x86/index.html`, which doesn't exist.

Reply:
I think changing the links to point to `../../core/arch/x86/index.html` should fix this?

Reply:
I think so, yes. I was about to say that might break the docs of the `core_arch` crate itself, but it looks like those are already broken in the same way.